packages/rocketchat-e2e/client/rocketchat.e2e.js

```diff
@@ -165,7 +167,7 @@ class E2E {
 							return;
 						}
 						localStorage.removeItem('e2e.randomPassword');
-						alerts.close();
+						self.closeAlert();
```

Comment:
You could change the `action() {` to `action: () => {` then you will not need the `self` var